repository.rosinstall

```diff
@@ -0,0 +1,4 @@
+- git:
+    local-name: shadow_ros_control
+    uri: https://github.com/shadow-robot/ros_control.git
+    version: combined_robot_hw
```

Comment:
rosinstall should also include:

- https://github.com/ros-controls/ros_controllers.git     version `jade-devel`
- https://github.com/shadow-robot/ros_ethercat.git     version `F#41_combined_robot_hw`